torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -136,6 +136,14 @@ Node* addDummyClone(
       orig_data->type()->kind() == TypeKind::BoolType) {
     auto* noneNode = graph->create(prim::Constant);
     noneNode->output()->setType(NoneType::get());
+    // For scripting mode, aten::clone requires input to be a TensorType
+    // Hence if we encounter an IntType or loatType, we set the input
+    // to the appropriate TensorType
+    if (orig_data->type()->kind() == TypeKind::IntType) {
+      orig_data->setType(TensorType::fromNumberType(*IntType::get()));
+    } else if (orig_data->type()->kind() == TypeKind::FloatType) {
+      orig_data->setType(TensorType::fromNumberType(*FloatType::get()));
+    }
```

Comment:
Do we have to care about int8, float16, float64 or IntType::get()/FloatType::get() already handle these cases?

Reply:
Looks like [`fromNumberType`](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/jit_type.h#L1680-L1691) will use `at::kLong` and `at::kDouble`, respectively. The few other places that call it [do not seem to care](https://github.com/pytorch/pytorch/blob/fa09099ba35fcd42347732ca3a5f8ddaf145da1b/torch/csrc/jit/passes/erase_number_types.cpp#L12-L18) about [other type sizes](https://github.com/pytorch/pytorch/search?q=fromNumberType).